docresources/SyncBot/syncbot-readme.md

```diff
@@ -18,7 +128,7 @@ A little Bot, using the ProcBot framework, that links communication services
     * Uses front's `comment`
     * Without syntax, private is assumed
 
-## Your Configuration
+### Your Configuration
 
 * Ensure your Front, Discourse & Flowdock usernames match
```

Comment:
Do we now have a hardlink for the docs you wrote for process? If so, I'd suggest adding the link here.